### PR TITLE
Fixes #17940: CVE plugin api documentation

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://issues.rudder.io/issues/17940

### CODE REVIEW

**Comment by VinceMacBuche** [comment]
PR updated with a new commit

**Comment by VinceMacBuche** [comment]
PR updated with a new commit

File: webapp/sources/api-doc/components/schemas/cveCheck.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  packages:
+    type: array
+    description: packages affected by this CVE
+    items:
+      type: object
+      description: "Packages affected by this CVE"
+      properties:
+        name:
+          type: string
+          description: Name of the package affected
```
Comment (amousset) [inline webapp/sources/api-doc/components/schemas/cveCheck.yml:25]: could you add example for these two values?

**Comment by VinceMacBuche** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**